app/webpacker/components/ContactsPage/ContactForm.jsx

```diff
@@ -66,6 +68,7 @@ export default function ContactForm({ loggedInUserData }) {
   }, [selectedContactRecipient]);
 
   if (saving) return <Loading />;
+  if (saveError) return <Errored error={saveError} />;
```

Comment:
I think the error handler of our `useSaveAction` returns the actual error _object_. This means that the `Errored` property here might end up just displaying: "An error occured! Details: [Object object]".

Have you tested this? You can temporarily modify the backend controller to just execute `raise "Something went horribly wrong!"` directly as the first line, to simulate an error. And then see how the frontend behaves.

Reply:
Yes, it returns error object. If I do `raise "Something went horribly wrong!"`, it will show some weird error:
<img width="1401" alt="Screenshot 2024-06-24 at 7 06 57 AM" src="https://github.com/thewca/worldcubeassociation.org/assets/9512698/66440108-457b-4c1f-a2cf-5e09fe7b43ec">

But instead for `return render status: :bad_request, json: { error: "Invalid arguments" }`, I'll get following:
<img width="1404" alt="Screenshot 2024-06-24 at 7 08 11 AM" src="https://github.com/thewca/worldcubeassociation.org/assets/9512698/ba009a28-7147-4929-af32-c3f8205dfbe6">

The last one is the current behavior. Is this fine or do you have an alternative suggestion?

(I also just realized I missed to add `return` for that error in contact_controller. Added it now)